alembic/versions/af5c30d91685_remove_project_id_task_queue.py

```diff
@@ -0,0 +1,51 @@
+"""remove project id task queue
+
+Revision ID: af5c30d91685
+Revises: 0d587af700ce
+Create Date: 2024-08-04 15:42:26.747671
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'af5c30d91685'
+down_revision = '0d587af700ce'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('macro_execution_summary',
+    sa.Column('id', postgresql.UUID(as_uuid=True), nullable=False),
+    sa.Column('organization_id', postgresql.UUID(as_uuid=True), nullable=True),
+    sa.Column('creation_month', sa.Date(), nullable=True),
+    sa.Column('macro_type', sa.String(), nullable=True),
+    sa.Column('execution_count', sa.Integer(), nullable=True),
+    sa.Column('processed_files_count', sa.Integer(), nullable=True),
+    sa.ForeignKeyConstraint(['organization_id'], ['organization.id'], ondelete='CASCADE'),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('organization_id', 'creation_month', 'macro_type', name='unique_macro_summary'),
+    schema='cognition'
+    )
+    op.create_index(op.f('ix_cognition_macro_execution_summary_creation_month'), 'macro_execution_summary', ['creation_month'], unique=False, schema='cognition')
```

Comment:
looks like it's still in the refinery schema. We wanted to change this to global

Reply:
(which should be more of a drop old table, create new table)